src/main/pages/che-7/hands-on-guidance/proc_adding-support-for-a-new-language.adoc

```diff
@@ -1,27 +1,39 @@
 [id="adding-support-for-a-new-language_{context}"]
 = Adding support for a new language
 
-This paragraph is the procedure module introduction: a short description of the procedure.
+This section describes how to add support for a new language into Eclipse Che. 
+
+While you can directly hook up a Theia plugin or VS Code extension to the plugin API, the usual way is to write a VS Code extension that uses the languageserver-node library to connect a language server to the VS Code extension API.
 
 [discrete]
 == Prerequisites
 
-* A bulleted list of conditions that must be satisfied before the user starts following this assembly.
-* You can also link to other modules or assemblies the user must follow before starting this assembly.
-* Delete the section title and bullets if the assembly has no prerequisites.
+* Install the latest Che-Theia plug-in Yeoman generator by running:
++
+----
+$ npm install -g yo @theia/generator-plugin
+----
 
 [discrete]
 == Procedure
 
-. Start each step with an active verb.
+. Generate a sample by running `yo @theia/plugin`. When prompted, confirm the choices for *Backend plug-in*, *Samples*, and then *Language sample*. This creates a project and a ready-to-use plug-in, which can be deployed.
```

Comment:
Could this be pointing to a sample plugin for LS? I think that would help understand with some concrete examples.

Reply:
This generated sample already provides ready to use plug-in with XML Language Server